datacenter/ucp/2.0/guides/configuration/index.md

```diff
@@ -39,11 +39,13 @@ and click **Certificates**.
 
 Upload your certificates and keys:
 
-* A ca.pem file with the root CA public certificate.
-* A cert.pem file with the TLS certificate and any intermediate CA public
+* A `ca.pem` file with the root CA public certificate followed by any intermediate CA public 
```

Comment:
After several tests and a conversation with @programmerq I can confirm that this is what you should do:
* CA - the CA public certificate
* Server cert - The public certificate for your domain, and any intermediate public certificates, in this order.

Can you revert your change to line `42` and update `44` make make this clear?